data/json/items/comestibles/vegetable.json

```diff
@@ -0,0 +1,312 @@
+    "weight": 170,
+    "volume": 1,
+    "price": 400,
+    "spoils_in": 160,
+    "nutrition": 30,
+    "healthy": 2,
+    "flags": [ "EATEN_HOT" ]
+  },
+  {
+    "id": "buckwheat",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "buckwheat",
+    "name_plural": "buckwheat",
+    "description": "Seeds from a wild buckwheat plant.  Not particularly good to eat in their raw state, they are commonly cooked or ground into flour.",
+    "weight": 144,
+    "volume": 2,
+    "price": 90,
+    "charges": 2,
+    "nutrition": 22,
+    "quench": -8,
+    "healthy": 1,
+    "fun": -10
+  },
+  {
+    "id": "cabbage",
+    "copy-from": "vegetable_leafy",
+    "type": "COMESTIBLE",
+    "name": "cabbage",
+    "description": "A hearty head of crisp white cabbage.",
+    "weight": 235,
+    "volume": 1,
+    "price": 100,
+    "spoils_in": 240,
+    "nutrition": 15,
+    "quench": 2,
+    "healthy": 1,
+    "fun": -2
+  },
+  {
+    "id": "carrot",
+    "copy-from": "vegetable_root",
+    "type": "COMESTIBLE",
+    "name": "carrot",
+    "name_plural": "carrots",
+    "description": "A healthy root vegetable.  Rich in vitamin A!",
+    "weight": 80,
+    "volume": 1,
+    "price": 200,
+    "charges": 6,
+    "nutrition": 5,
+    "healthy": 1,
+    "fun": -1,
+    "proportional": { "vitamins": [ "VitA", 3.0 ] },
+    "relative": { "spoils_in": 0.5 }
+  },
+  {
+    "id": "cattail_rhizome",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "cattail rhizome",
+    "name_plural": "cattail rhizomes",
+    "description": "A stout branching rhizome from a cattail plant.  Its crisp white flesh is very starchy and fibrous, but you really ought to cook it before you attempt to eat it.",
+    "weight": 418,
+    "volume": 2,
+    "price": 0,
+    "spoils_in": 80,
+    "nutrition": 22,
+    "fun": -12
+  },
+  {
+    "id": "cattail_stalk",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "cattail stalk",
+    "name_plural": "cattail stalks",
+    "description": "A stiff green stalk from a cattail plant.  It is starchy and fibrous, but it would be much better if you cooked it.",
+    "weight": 146,
+    "volume": 1,
+    "price": 0,
+    "spoils_in": 80,
+    "nutrition": 5,
+    "fun": -8
+  },
+  {
+    "id": "celery",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "celery",
+    "description": "Neither tasty nor very nutritious, but it goes well with salad.",
+    "weight": 80,
+    "volume": 1,
+    "price": 200,
+    "spoils_in": 150,
+    "nutrition": 5,
+    "quench": 3
+  },
+  {
+    "id": "cucumber",
+    "copy-from": "vegetable",
+    "type": "COMESTIBLE",
+    "name": "cucumber",
+    "description": "Come from the gourd family, not tasty but very juicy.",
+    "weight": 150,
+    "volume": 1,
+    "price": 300,
+    "spoils_in": 150,
+    "nutrition": 11,
+    "quench": 7,
+    "healthy": 1
+  },
+  {
+    "id": "garlic",
```

Comment:
Note: there are two garlics - one is just 1/6 of the other. It would be good to merge them into one item (preferably the more granular one, since no sane person will eat a whole bulb of garlic in one go).